apps/publisher/modules/lifecycle/lifecycle-api.js

```diff
@@ -409,6 +409,13 @@ var error = '';
         state.isLCActionsPermitted = isLCActionsPermitted(asset, options, req, res, session);
         state.checkItems = setCurrentCheckItemState(state.checkItems, lcCheckedStates, state.isLCActionsPermitted);
         state.approvedActions = setAvailableApprovedActions(state.approvedActions, lcCheckedStates);
+        var isStateDurationEnable = rxtManager.isLCStateDurationStateEnabled(options.type);
```

Comment:
Eagerly set state.isLCStateDurationEnabled = false;

Then we can get rid of the else statement